app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all.order(:name)
```

Comment:
`.order` is an AR method that queries the db, which we don't want to do in the controller given that it's beyond a simple CRUD operation. A more MVC-compliant option would be to create a class method in the model to do the `order` operation and call that class method here